Tools/LambdaTestTool/tests/Amazon.Lambda.TestTool.Tests.NET6/LambdaExecutorTests.cs

```diff
@@ -0,0 +1,173 @@
+using Amazon.Lambda.TestTool.Runtime;
+using Amazon.Lambda.TestTool.Runtime.LambdaMocks;
+
+namespace Amazon.Lambda.TestTool.Tests.NET6
```

Comment:
I've added these to the .NET6 namespace as I'm unable to properly test them running on an ARM MacBook and don't have access to an x86 machine where I can properly install the 3.1 runtime